### PR TITLE
Successfully done the grid alignment of HR resources

### PR DESCRIPTION
**Pull request by shardulgusain**
#1839

### Description
Some small bootstrap changes are made as per the requirements.
### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by P4NK4J** [comment]
@shardulgusain Can we shift the whole container, including title and cards to the center of the page
image

Also, can we increase the margin of the title and buttons from the header
image

Please make the above changes and assign for review once done.

**Comment by shardulgusain** [comment]
> Left a comment, please check.

"Can we shift the whole container, including title and cards to the center of the page"

Do you want me to increase left margin of the entire container....or I have to shift the entire heading to the center of the page ?

**Comment by P4NK4J** [comment]
> > Left a comment, please check.
> 
> "Can we shift the whole container, including title and cards to the center of the page"
> 
> Do you want me to increase left margin of the entire container....or I have to shift the entire heading to the center of the page ?

I am assuming the heading and the cards must be inside a master container, we need to increase the margin from left and right, for that to align in center.

File: Modules/HR/Resources/views/guidelines-resources/show.blade.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
     </div>
 </div>
 <div class="container mx-15 mt-5">
-    <p>No resource exist for this job. </p>
+    <div class="text-center mt-25">No resource exist for this job. </div>
```
Comment (P4NK4J) [inline Modules/HR/Resources/views/guidelines-resources/show.blade.php:19]: Text inside `<div>` is not the best approach from SEO point of view. I would use `<div>` wherever need to add a space block, container etc.

**Review by P4NK4J — APPROVED** [review]
Left a minor comment, else everything looks good.

File: Modules/HR/Resources/views/guidelines-resources/show.blade.php (hunk 1 above)
Comment (shardulgusain) [reply Modules/HR/Resources/views/guidelines-resources/show.blade.php]: Noted and Done.

**Pull request merged**